src/avfoundation.cs

```diff
@@ -3123,7 +3123,7 @@ interface AVCaptureSynchronizedDataCollection
 
 	interface IAVCaptureDataOutputSynchronizerDelegate {}
 	
-	[NoWatch, NoTV, iOS (11,0)]
+	[NoWatch, NoTV, iOS (11,0), NoMac]
```

Comment:
that's a breaking change :|
but I see why it does not work
which is 50% good news for such mistakes